file_paths.md

```diff
@@ -292,6 +300,14 @@ create a `puppet` user or group.
             puppet *                          # ruby bin wrapper
         lib *
             puppet.rb *
+        share *
+            locale
```

Comment:
Anything created at installation time should have a `*` next to it.

A lack of a `*` generally indicates Puppet creates (and typically manages) the file / directory on first run.  I believe this entire path structure is created by installing a package (for both Windows and Linux)